src/icareFhirMessaging.js

```diff
@@ -108,6 +108,10 @@ async function postExtractedData(messagingClient, bundledData) {
   bundledData.forEach(async (bundle, i) => {
     messagingErrors[i] = [];
     try {
+      if (bundle && bundle.entry[1].resource.entry.length === 0) {
+        logger.warn(`No data was extracted for patient at row ${i + 1}, message will not be sent for this patient`);
+        return;
+      }
```

Comment:
I think this warning makes a lot of sense! I know we can't test logged messages yet, but is it possible to add a test to check that `processMessage` isn't called in this case? If that's not an easy thing to do, we don't need a test!

Reply:
This is definitely possible and also a great thing to test! I'll add that in as well.